main/src/CoffeeMachineSimulator.java

```diff
@@ -0,0 +1,200 @@
+import java.util.Scanner;
+
+public class CoffeeMachineSimulator {
+
+    //declaring our class variables
+    int water = 400;
+    int milk = 540;
+    int coffeeBeans = 120;
+    int disCups = 9;
+    int money = 550;
+    String buy = "buy";
+    String fill = "fill";
+    String take = "take";
+    String exit = "exit";
+    String remaining = "remaining";
+    String back = "back";
+    int mode = 1;
+
+
+    // method myMethod check coffee machine mode, and call appropriate method
+    public void myMethod(String inputString) {
```

Comment:
it's not good ida to set method name like this, try to set same meaning names checkMode for example

Reply:
done